wandb/media.py

```diff
@@ -0,0 +1,81 @@
+import os
+from wandb import util
+
+
+class Media(object):
+    @staticmethod
+    def from_type(type):
```

Comment:
What's from_type?

If it's what I'm guessing it's probably better as a classmethod, which gives you the class as an argument (with a static method if you're generating instances of your class, you have to hardcode the class name within the class itself, which doesn't play as well with inheritance).